src/cloudinventario_amazon_aws/res_collectors/elb.py

```diff
@@ -0,0 +1,74 @@
+import boto3, json
+from pprint import pprint
+
+from cloudinventario.helpers import CloudInvetarioResource
+
+def get_resource_obj(credentials):
+  return CloudInventarioElb(credentials)
+
+class CloudInventarioElb(CloudInvetarioResource):
+
+  def __init__(self, credentials):
+    super().__init__("elb", credentials)
+
+  def _get_client(self):
+    client = boto3.client('elb', aws_access_key_id = self.credentials["access_key"], aws_secret_access_key = self.credentials["secret_key"],
+                                  aws_session_token = self.credentials["session_token"], region_name = self.credentials["region"])
+    return client
+
+  def _fetch(self):
+    data = []
+    paginator = self.client.get_paginator('describe_load_balancers')
+
+    for page in paginator.paginate():
+      page = self.client.describe_load_balancers(PageSize=100)
+
+      for lb in page['LoadBalancerDescriptions']:
+        data.append((self._process_resource(lb), lb))
+      
+      next_marker = None
+      if 'NextMarker' in page:
+         next_marker = page['NextMarker']
+      if not next_marker:
+        break
```

Comment:
I believe here are being mixed paginator concept with NextToken/NextMarker functionality.
Although the paginator is created and iterated, in next step is paginated `page` is overwriten with call to describe_load_balancers. This seems not to be correct.

See https://stackoverflow.com/questions/39201093/how-to-use-boto3-pagination#39206941
or https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html